core/src/tangram.cpp

```diff
@@ -587,6 +603,34 @@ void Map::clearDataSource(DataSource& _source, bool _data, bool _tiles) {
     requestRender();
 }
 
+Marker* Map::markerAdd(const char* _styling) {
+    return impl->markerManager.add(_styling);
+}
+
+bool Map::markerRemove(Marker* _marker) {
+    return impl->markerManager.remove(_marker);
+}
+
+bool Map::markerSetPoint(Marker* _marker, LngLat _lngLat) {
+    return impl->markerManager.setPoint(_marker, _lngLat);
+}
+
+bool Map::markerSetPointEased(Marker* _marker, LngLat _lngLat, float _duration, EaseType ease) {
+    return impl->markerManager.setPointEased(_marker, _lngLat, _duration, ease);
+}
+
+bool Map::markerSetPolyline(Marker* _marker, LngLat* _coordinates, int _count) {
```

Comment:
Should these `markerSetGeom` methods be doing a `requestRender` in the core, or should the client application be responsible to do that `requestRender` after setting geometry for a marker.

Currently I am doing a `requestRender` from tizen plugin code, on update of a marker.